js/everything.js

```diff
@@ -1,3 +1,18 @@
+// DarkTheme Function:
+
+var modeIcon = document.getElementById("modeIcon");
+
+modeIcon.onclick = function(){
+    
+    document.body.classList.toggle("dark-theme");
+
+    if(document.body.classList.contains("dark-theme")){
+        modeIcon.src = "img/icons/brightness_5_FILL1_wght700_GRAD0_opsz48.png";
```

Comment:
We should be using fontawesome icons instead of these new images, let’s start there.

Reply:
Ok, I'm on it